frameworks.md

```diff
@@ -81,11 +87,9 @@ Modules particularities:
   * the start and end of a module, even the global one, are emitted only if the suite itself contains at least one test
   * nested modules have a concatenated name, from the outer most suite to the inner most
 
-Nested modules execution is done the following way:
-  * take in order the modules inside the top level module
-  * than execute them from the inner most module to the outer most
+The execution is done in the source order, but QUnit has a more flat style for nested modules, it emit the start of a module, emit its tests, then the module ends and starts another, even if the modules were nested, there is not a sort of recursion, like in the CRI standard.
```

Comment:
Let's describe the framework independent of other frameworks. The reader should be able to skip directly to read about QUnit, instead of having to read about Mocha first.